troposphere/static/js/AppRoutes.react.js

```diff
@@ -66,22 +67,22 @@ define(function (require) {
       </Route>
 
       <Route name="providers" handler={ProvidersMaster}>
-        <Route name="provider" path=":providerId" handler={ProviderDetailsPage}/>
-        <DefaultRoute handler={ProviderDetailsPage}/>
+        <DefaultRoute handler={ProviderListSection} />
+        <Route name="provider" path=":id" handler={ProviderDetail}/>
+        <Route name="all-providers" path="/" handler={ProviderListSection} />
       </Route>
 
       <Route name="help" handler={HelpPage}/>
       <Route name="settings" handler={SettingsPage}/>
 
       <Route name="admin" handler={AdminMaster}>
-        <Route name="identity-membership-manager" path="users" handler={IdentityMembershipMaster}/>
+        <DefaultRoute name="identity-membership-manager" path="users" handler={IdentityMembershipMaster}/>
```

Comment:
I think I intentionally changed it to DefaultRoute but I should have removed the path. I don't know how it leaked back in to my branch but I should have caught it.

This is something we should look at however. The pattern here on the Admin Page exposes a similar architecture/ user issue we had with Providers. We "think" we want the IdentityMembership to be DefaultRoute but we also want it to be on a sub-path of Admin. So like with Admin to we redirect the user into users or does Admin need to have a view of it's own?